home/templates/includes/navigation.html

```diff
@@ -188,71 +188,55 @@
+          <li class="nav-item dropdown1">
             <a
               href="#"
               class="nav-link dropdown-toggle"
-              id="frontPagesDropdown"
+              id="projectsDropdown"
               aria-expanded="false"
-              data-bs-toggle="dropdown"
             >
               Projects
               <span class="fas fa-angle-down nav-link-arrow ms-1"></span>
             </a>
-            <div
-              class="dropdown-menu px-0 py-2 p-lg-4"
-              aria-labelledby="frontPagesDropdown"
-            >
+            <div class="dropdown-menu1 px-0 py-2 p-lg-4" aria-labelledby="projectsDropdown">
               <div class="row">
                 <div class="col">
                   <ul class="list-style-none mb-4">
                     <li class="mb-2 menu-item">
-                      <a class="menu-link" href="{% url 'appattack' %}"
-                        >AppAttack</a
-                      >
+                      <a class="menu-link" href="{% url 'appattack' %}">AppAttack</a>
                     </li>
                     <li class="mb-2 menu-item">
-                      <a class="menu-link" href="{% url 'malware_viz_main' %}"
-                        >Malware (On Hold)</a
-                      >
+                      <a class="menu-link" href="{% url 'malware_viz_main' %}">Malware (On Hold)</a>
                     </li>
                     <li class="mb-2 menu-item">
-                      <a class="menu-link" href="{% url 'ptgui_viz_main' %}"
-                        >PT-GUI</a
-                      >
+                      <a class="menu-link" href="{% url 'ptgui_viz_main' %}">PT-GUI</a>
                     </li>
-
-
                     <li class="mb-2 menu-item">
-                      <a
-                        class="menu-link"
-                        href="{% url 'smishing_detection_main' %}"
-                        >Smishing Detection</a
-                      >
+                      <a class="menu-link" href="{% url 'smishing_detection_main' %}">Smishing Detection</a>
                     </li>
-
-                
-                  <li class="mb-2 menu-item">
+                    <li class="mb-2 menu-item">
                       <a class="menu-link" href="{% url 'Vr_main' %}">Deakin CyberSafe VR</a>
-                  </li>
-                  <li class="mb-2 menu-item">
+                    </li>
+                    <li class="mb-2 menu-item">
                       <a class="menu-link" href="{% url 'Deakin_Threat_mirror_main' %}">Deakin Threat mirror (On Hold)</a>
-                  </li> 
-
+                    </li>
                   </ul>
                 </div>
               </div>
             </div>
           </li>
-
-
-          <li class="nav-item dropdown">
-            <a href="#" class="nav-link dropdown-toggle" id="frontPagesDropdown" aria-expanded="false"
-              data-bs-toggle="dropdown">
+          
+          <li class="nav-item dropdown1">

```

Comment:
use "nav-item dropdown" instead of "nav-item dropdown1"

Reply:
the class with the name of "nav-item dropdown" is a built in class which conflicts with my current functionality, so I used this class to differentiate from conflict.